content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,209 @@
 
-Let’s suppose there are one warehouse location *WH/Stock* and two sub-locations
-*WH/Stock/Vegetables* and *WH/Stock/Fruits*.
+.. example::
+   Ensure all apples are stored in the fruits section by filling field :guilabel:`Store to` with the
+   location `WH/Stock/Fruits` when the :guilabel:`Product`, `Apple` arrives in `WH/Stock`. Repeat
+   this for all products and hit :guilabel:`Save`.
 
-To manage those locations, we will create putaway rules. To do so, open the *Inventory* app and go
-to :menuselection:`Configuration --> Putaway Rules`. Then, click on create and configure your first
-rule indicating the main location the product will enter before being redirected to the right
-location.
+   .. image:: putaway/create-putaway-rules.png
+      :align: center
+      :alt: Create putaway rules for apples and carrots.
 
 .. note::
-   The putaway rules can be defined either per product or per product category.
+  It is also possible to create and manage putaway rules for a single product by going to the
+  product page and clicking the :guilabel:`Putaway Rules` smart button at the top of the page. If
+  the button isn't there at first glance, select the :guilabel:`More` button at the top right to
+  view additional configuration options.
+
+.. _inventory/putaway/internal-stock-moves:
+
+.. tip::
+   Check whether putaway rules work as intended and using the summary of internal product movements.
+   In the :guilabel:`Inventory` app, select :menuselection:`Reporting --> Product Moves` and enable
+   the :guilabel:`Internal` search filter on the :guilabel:`Filters` dropdown under the search bar
+   at the top of the page. In this view, confirm that products are moved to the :guilabel:`Store to`
+   location detailed in the putaway rule. More details in :ref:`this section
+   <inventory/putaway/wrong-location>`.
+
+Use putaway rules
+-----------------
+
+Once a putaway rule has been configured, the product it specifies is automatically routed to the
+:guilabel:`Store to` location upon arriving in the :guilabel:`When product arrives in` location.
+
+After configuring the putaway rules for apples and carrots, see the internal stock moves by
+first buying products from a vendor using a :ref:`purchase order<inventory/purchase/vendor bills>`
+(PO).
 
-.. image:: putaway/putaw2.png
+To create a :guilabel:`Request for Quotation` (RFQ), go to the :guilabel:`Purchase` app, and click
+:guilabel:`Create`. Then, add products to the :abbr:`RFQ (Request for Quotation)` by clicking
+:guilabel:`Add a line`, and selecting the desired product(s) from the drop-down menu in the
+:guilabel:`Product` column.
+
+Once ready, click :guilabel:`Confirm`. Doing so turns the :abbr:`RFQ (Request for Quotation)` into a
+:abbr:`PO (Purchase Order)`. Next, click :guilabel:`Send by Email` to send the :abbr:`PO (Purchase
+Order)` to the vendor if needed.
+
+.. image:: putaway/purchase-apples-and-carrots.png
    :align: center
+   :alt: Purchase order for apples and carrots
```

Comment:
It's not super clear why there's a red box around the "Receipt" smart button in this image. You might want to add a sentence right before this explaining the smart button.